JS/fluentRevealNavbar.uc.js

```diff
@@ -16,14 +16,28 @@
       // if true, show the effect on bookmarks on the toolbar
       includeBookmarks: true,
 
+      // if true, show the effect on the urlbar
+      includeUrlBar: true,
+
       // the color of the gradient. default is sort of a faint baby blue. you may prefer just white, e.g. hsla(0, 0%, 100%, 0.05)
-      lightColor: "hsla(224, 100%, 80%, 0.15)",
+      // lightColor: "hsla(224, 100%, 80%, 0.15)",
+
+      // the color of the gradient. default is the browser's color on navbar button hover, or a faint baby blue if it's not available. 
+      // you may prefer just white, e.g. hsla(0, 0%, 100%, 0.05)
+      lightColor: "var(--button-hover-bgcolor, hsla(224, 100%, 80%, 0.15))",
 
       // how wide the radial gradient is.
       gradientSize: 50,
 
       // whether to show an additional light burst when clicking an element. (not recommended)
       clickEffect: false,
+
+      // don't process mouse movements greater than {gradientSize}px from top of the screen, in order to reduce system load.
+      // disable if you modified the ui to have toolbar buttons on different side of the screen (left, right or bottom)
+      filterDy: true,
```

Comment:
Think this should probably be false by default, given that it will break the script for users with some CSS customizations. It also will not work with some of my other scripts.